### DISCUSSION TITLE
Different Flashlight Colors

### DISCUSSION THREAD

**Discussion by 42Tobias42** [InfiniTimeOrg/InfiniTime#1962] [Ideas]
It was implemented in #633 but I want to reiterate on it.

I was sailing recently and a was searching for a red flashlight.
Different to #633 I was trying to change the color by long pressing the active flashlight screen.

I think that in combination with an indicater what color mode is active, when the flashlight is turned off would come in handy.

**Comment by NitroNils** [comment]
Or a global "night vision" mode, as has been discussed earlier.
